pkgs/tools/virtualization/dockstarter/default.nix

```diff
@@ -0,0 +1,43 @@
+{ bash
+, coreutils
+, fetchFromGitHub
+, git
+, lib
+, makeWrapper
+, ncurses
+, stdenv
+, sudo
+}:
+stdenv.mkDerivation rec {
+  pname = "dockstarter";
+  version = "unstable-2022-10-26";
+
+  src = fetchFromGitHub {
+    owner = "ghostwriters";
+    repo = pname;
+    rev = "a1b6b6e29aa135c2a61ea67ca05e9e034856ca08";
+    hash = "sha256-G26DFme6YaizdE5oHBo/IqV+1quu07Bp+IykXtO/GgA=";
+  };
+
+  doBuild = false;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    install -Dm755 main.sh $out/bin/ds
+    wrapProgram $out/bin/ds --prefix PATH : ${lib.makeBinPath ((lib.optionals (!stdenv.isDarwin) [ sudo ]) ++ [
+      bash
```

Comment:
Is this execing bash? If it is a shebang we should fix it otherwise.

Reply:
yeah, it is a bit of an oddball: https://github.com/GhostWriters/DockSTARTer/blob/master/main.sh#L400